bridge/client/app/_components/ktb-evaluation-details/ktb-evaluation-details.component.ts

```diff
@@ -320,10 +320,17 @@ export class KtbEvaluationDetailsComponent implements OnInit, OnDestroy {
     private dialog: MatDialog,
     private clipboard: ClipboardService,
     public dateUtil: DateUtil,
-    private zone: NgZone
+    private zone: NgZone,
+    private featureFlagService: FeatureFlagsService
   ) {}
 
   public ngOnInit(): void {
+    this.featureFlagService.featureFlags$
```

Comment:
I'd prefer the reactive approach here:

```suggestion
    // outside of constructor
    public d3HeatmapEnabled$ = this.featureFlagService.featureFlags$
          .pipe(map((featureFlags: IClientFeatureFlags) => featureFlags.D3_HEATMAP_ENABLED));
```

No need for unsubscribe and have a separate state in the component.